core/src/main/java/com/gentics/mesh/core/endpoint/admin/AdminEndpoint.java

```diff
@@ -305,7 +305,7 @@ protected void addDebugInfoHandler() {
 		route.path("/debuginfo");
 		route.method(GET);
 		route.description("Downloads a zip file of various [debug information](/docs/administration-guide/#debuginfo) files.");
-		route.addQueryParameter("include", "Information to include. See the [documentation](/docs/administration-guide/#debuginfo) for possible values.", "-backup,consistencyCheck");
+		route.addQueryParameter("include", "Information to include. See the [documentation](/docs/administration-guide/#debuginfo) for possible usages.", "-log,consistencyCheck");
```

Comment:
Since we have no more `Reflections`, it is either not to hint for the possible values (which weren't served anyway) or to have a static value list just in the docu (not forgetting to update it, if something changes...).